engine/src/main/java/org/terasology/engine/core/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -48,8 +47,8 @@
     public HeadlessWorldRenderer(Context context) {
         this.worldProvider = context.get(WorldProvider.class);
         this.chunkProvider = context.get(ChunkProvider.class);
-        LocalPlayerSystem localPlayerSystem = context.get(LocalPlayerSystem.class);
-        localPlayerSystem.setPlayerCamera(noCamera);
+//        LocalPlayerSystem localPlayerSystem = context.get(LocalPlayerSystem.class);
+//        localPlayerSystem.setPlayerCamera(noCamera);
```

Comment:
Oh, sure, I guess a headless system wouldn't have a local player, would it?

Reply:
Yeah. It should not.
Mte headless host = networkmode.listen-server
There isLocalClient is false :)

Seems headless version was problem with camera and input system